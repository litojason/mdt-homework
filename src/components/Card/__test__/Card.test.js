import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import { Card, Row } from "../index";

describe("Component: Card", () => {
  it("should render Card", () => {
    const component = <Card />;

    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should render Row", () => {
    const component = <Row />;

    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
