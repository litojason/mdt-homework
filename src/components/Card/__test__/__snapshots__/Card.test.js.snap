// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Card should render Card 1`] = `
.c0 {
  margin-bottom: 1rem;
  padding: 1rem;
  background-color: #FFF;
  border-radius: 0.5rem;
  box-shadow: 0px 3px 6px 0px rgba(0,0,0,0.125);
}

<div
  className="c0"
/>
`;

exports[`Component: Card should render Row 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin-bottom: 0.25rem;
}

.c0:last-child {
  margin-bottom: 0;
}

<div
  className="c0"
/>
`;
